src/elements/hx-toggle/index.js

```diff
@@ -0,0 +1,28 @@
+import { HXFormControlElement } from '../../interfaces/HXFormControlElement/index.js';
+
+import shadowMarkup from './_shadow.html';
+import shadowStyles from './_shadow.scss';
+
+/**
+ * Defines behavior for the `<hx-toggle>` element.
+ * NOTE: `<hx-toggle>` can have various default options, or it can be overriden.
```

Comment:
Please remove this line in the comments.  This refers to the Switch component.

Reply:
Changes have been done, please review.